### PR TITLE
[staging] fix: fixed header tab ui [123]

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

In this PR i have added curve in the selected tab in header tab

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@workspaces/features/tab-bar/components/tab/Tab.svelte [hunk 1]
```diff
@@ -196,4 +207,23 @@
...
+    background-color: var(--text-secondary-800);
+  }
+  .box1 {
+    position: absolute;
+    left: -9px;
+    border-radius: 70px;
+    bottom: -20px;
+    background-color: var(--text-secondary-800);
+  }
+  .btn-global {
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/tab-bar/components/tab/Tab.svelte:226]: set zIndex to around 500

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**